config/default.yml

```diff
@@ -1598,6 +1598,13 @@ Lint/SafeNavigationChain:
     - presence
     - try
 
+Lint/SafeNavigationConsistency:
+  Whitelist:
+    - present?
+    - blank?
+    - presence
+    - try
```

Comment:
Can you also include `try!` method in the `Whitelist`?
http://api.rubyonrails.org/classes/Delegator.html#method-i-try-21

Reply:
Good call. I'll add `try!` to all of the safe navigation cops' whitelist.